stdlib/time.pyi

```diff
@@ -80,7 +79,13 @@ def time() -> float: ...
 if sys.platform != "win32":
     def tzset() -> None: ...  # Unix only
 
-def get_clock_info(name: str) -> SimpleNamespace: ...
+class _ClockInfo(Protocol):
+    adjustable: bool
+    implementation: str
+    monotonic: bool
+    resolution: float
+
+def get_clock_info(name: Literal["monotonic", "perf_counter", "process_time", "time"]) -> _ClockInfo: ...
```

Comment:
CPython tests also call this with `"thread_time"`. Looking at the C source code, that's the only missing name.

Reply:
Looks like it is also missing from the docs 🤔 
I will investigate! Thank you!

Reply:
Filled https://bugs.python.org/issue46530